tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -3030,8 +3043,12 @@ static unsigned AllocateDxilConstantBuffer(HLCBuffer &CB) {
 
     unsigned size = C->GetRangeSize();
     llvm::Type *Ty = C->GetGlobalSymbol()->getType()->getPointerElementType();
+    auto fieldAnnotation = constVarAnnotationMap[C->GetGlobalSymbol()];
```

Comment:
prefer `.at` to error on the key-not-found case.

Reply:
Changed.